pkg/sdk/system_functions.go

```diff
@@ -37,6 +40,20 @@ func (c *systemFunctions) GetTag(ctx context.Context, tagID ObjectIdentifier, ob
 	return s.Tag, nil
 }
 
+// normalize object types for some values because of errors like below
+// SQL compilation error: Invalid value VIEW for argument OBJECT_TYPE. Please use object type TABLE for all kinds of table-like objects.
+// TODO [SNOW-1022645]: discuss how we handle situation like this in the SDK
+func normalizeGetTagObjectType(objectType ObjectType) ObjectType {
+	if slices.Contains([]ObjectType{ObjectTypeView, ObjectTypeMaterializedView, ObjectTypeExternalTable}, objectType) {
```

Comment:
what about dynamic table?

Reply:
Initially, I missed dynamic tables because they are missing from https://docs.snowflake.com/en/user-guide/object-tagging. Managing dynamic tables is not implemented in the SDK. I just checked it manually, and it turns out you can use GetTag with `dynamic table` as an argument.

Reply:
Good idea, next PR/follow-up issue?